content/master/software/install.md

```diff
@@ -332,7 +332,7 @@ Below are vendors providing conformant Crossplane distributions.
 #### Upbound
 Upbound, the founders of Crossplane, maintains a free and open source 
 distribution of Crossplane called 
-[Universal Crossplane](https://www.upbound.io/products/universal-crossplane)
+[Universal Crossplane](https://www.upbound.io/product/universal-crossplane)
```

Comment:
this change looks correct to me now and is working on the preview site. nice find!

can you also make this same change to https://github.com/crossplane/docs/blob/master/content/v1.16/software/install.md, so folks will see your update right away on the latest (i.e. v1.16) version of the site? 😁

Reply:
Included the fix in all the versions